packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -24,12 +50,12 @@ export default function register() {
                   // the fresh content will have been added to the cache.
                   // It's the perfect time to display a "New content is
                   // available; please refresh." message in your web app.
-                  console.log('New content is available; please refresh.');
+                  showMessage('New content is available. Please refresh.');
                 } else {
                   // At this point, everything has been precached.
                   // It's the perfect time to display a
                   // "Content is cached for offline use." message.
-                  console.log('Content is cached for offline use.');
+                  showMessage('Content is cached for offline use.');
```

Comment:
One possible addition would be to also include a "[learn more](https://github.com/facebookincubator/create-react-app/tree/master/packages/react-scripts/template#making-a-progressive-web-app)" link that pointed to the documentation for additional context. Using a timeout of longer than 3 seconds might be appropriate in that case.